src/themes/seher/variables.js

```diff
@@ -15,11 +15,11 @@ export const variables = {
 	verticalBase: '2.4rem',
 
 	headingSmallSize: '1.8rem',
```

Comment:
I see that small and medium headings have the same values. Is it intended?

Reply:
I guess so. @MiaTollaks tells me there's going to be a font change come January, I guess we'll revisit the font sizes then.

Reply:
Ok :+1: